lib/galaxy_test/selenium/test_workflow_editor.py

```diff
@@ -1052,17 +1052,22 @@ def workflow_editor_connect(self, source, sink, screenshot_partial=None):
             self.screenshot(screenshot_partial)
         drag_and_drop(self.driver, source_element, sink_element)
 
-    def assert_connected(self, source, sink):
-        source_id, sink_id = self.workflow_editor_source_sink_terminal_ids(source, sink)
-        self.components.workflow_editor.connector_for(source_id=source_id, sink_id=sink_id).wait_for_visible()
-
-    def assert_connection_invalid(self, source, sink):
-        source_id, sink_id = self.workflow_editor_source_sink_terminal_ids(source, sink)
-        self.components.workflow_editor.connector_invalid_for(source_id=source_id, sink_id=sink_id).wait_for_present()
-
-    def assert_not_connected(self, source, sink):
-        source_id, sink_id = self.workflow_editor_source_sink_terminal_ids(source, sink)
-        self.components.workflow_editor.connector_for(source_id=source_id, sink_id=sink_id).wait_for_absent()
+    def assert_connected(self, _source, _sink):
+        # NOOP
+        # Used to assert that an svg connection element exists.
+        # Since connections are no longer rendered with SVG, this is no longer possible.
+        # TODO: add different method of asserting connections
+        pass
+
+    def assert_connection_invalid(self, _source, _sink):
+        # NOOP
+        # TODO: add different method of asserting connections
```

Comment:
@mvdbeek I know how to replace these other two methods - is the correct way to replace this one checking there is a warning (via https://github.com/galaxyproject/galaxy/pull/15817) upon saving?